### PR TITLE
Sherwani/tidbits view added suggestion box

### PR DESCRIPTION
**Pull request by HussainKhanSherwani**
# Added a suggesetion box in rating for byte and guide

Before submitting this PR, please make sure you have completed the following checklist. This helps ensure that our codebase remains high quality and that our features work well across all supported devices and themes.

### Code Review
- [x] I have carefully reviewed my code and believe it is final from my side.
- [x] I have ensured there is no hardcoded color in my changes. Colors should be defined in theme files and reused.

### Theme Compatibility
- [x] I have tested the new feature or updates in both light and dark themes.
- [x] I confirm that the visual aspects of the feature are compatible and appealing in both themes.

### Device Compatibility
- [x] I have tested my changes on mobile screen sizes.
- [x] I have tested my changes on tablet screen sizes.
- [x] I have tested my changes on laptop screen sizes.

### Functional Testing
- [x] I have thoroughly tested the functionality of my changes.
- [x] I am confident that my changes do not negatively impact existing features.

### Testing Documentation
Please provide a brief description of the tests you have performed to verify your changes. Include any relevant scenarios or edge cases.

- Example test 1: Tested feature X by uploading the image
- Example test 2: Tested feature X by doing these things.

# Demo Video
Please provide a demo video of your changes. This helps reviewers understand the changes and verify the functionality.

You can record the video using tools like Loom(https://www.loom.com/), Clip by ClickUp(https://clickup.com/features/clips), Sendspark(https://www.sendspark.com/), etc.

Include the link to the video below and make sure the video covers
- [x] Explain the task/bug you are solving
- [x] Show the changes in the UI
- [x] Show the changes in the codebase
- [x] Show the changes you tested to make sure it works
- [x] Show the changes in various screen sizes
video URL:https://www.loom.com/share/0c254eae38f64e6d9f11b8c94b301bb5?sid=824a2fcc-d9bb-48ee-a230-f2c7b7f12360

### CODE REVIEW

File: shared/web-core/src/components/core/modals/FullScreenModal.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function FullScreenModal({ open, title, children, onClose, showCl
           <div className="h-screen bg-gray-500 bg-opacity-75 transition-opacity" />
         </Transition.Child>
 
-        <div className={`fixed inset-0 z-10 overflow-y-auto ${styles.modalContainer}`}>
+        <div className={`fixed inset-0 z-10${styles.modalContainer}`}>
```
[the remark this answers is not in the document]
Comment (HussainKhanSherwani) [reply shared/web-core/src/components/core/modals/FullScreenModal.tsx]: @MSamiTariq done with required changes you can review now

File: academy-ui/src/components/app/Modal/Rating/RatingModal.tsx [hunk 2]
```diff
@@ -28,6 +28,8 @@ export default function RatingModal<T extends GuideFeedback | ByteFeedback | und
   setRating,
 }: RatingModalProps<T>) {
   const [selectedRating, setSelectedRating] = useState<number>();
+  const [selectedFeedback, setSelectedFeedback] = useState<GuideFeedback | ByteFeedback>({});
```
Comment (MSamiTariq) [inline academy-ui/src/components/app/Modal/Rating/RatingModal.tsx:31]: I don't understand the use of UseState variable over here. Can you please elaborate?

Comment (HussainKhanSherwani) [reply academy-ui/src/components/app/Modal/Rating/RatingModal.tsx]: > I don't understand the use of UseState variable over here. Can you please elaborate?

it was not required and is removed

**Review by MSamiTariq — APPROVED** [review]
Looks good to me
